src/pages/en/guides/manual-setup.md

```diff
@@ -82,7 +82,22 @@ User-agent: *
 Allow: /
 ```
 
-## 5. Next steps
+## 5. Add a basic astro.config.mjs file into the project root
+
+Astro expects at least a minimal configuration file with the following minimal configuration content:
```

Comment:
Astro is configured using `astro.config.mjs` at the root of the project. This file is optional if you do not need to configure Astro, but you may wish to create it now. If you want to include framework components such as React, Svelte, etc. in your project, here is where you will [manually add any necessary renderers](en/core-concepts/framework-components/#customize-your-frameworks).

(Something like that?)